### DISCUSSION TITLE
Creating application with already existing resources

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#283] [Product Feedback]
### Which section?

Marketplace

### Which is the version?

11.3.0

### Description

When creating an application if there are existing resources (e.g. endpoints) there were going to be created with the application setup, the setup flow asks you to delete them before start to create the application.

I find this scenario very annoying and if possible it would be nice to be able to create the application and maintain the already existing resources
![image](https://github.com/mia-platform/community/assets/18056946/43061553-54e5-4d4a-a24b-99347acd82e8)

### Actual Outcome

Need to delete resources that are going to be created few steps later

### Expected Outcome

Keep the resources if related to existing microservices associated to the application

**Comment by fredmaggiowski** [comment]
Hi, as of now you can't create application requiring allocated endpoints.

The reason behind this limitation is that endpoints cannot be mapped to multiple services and if your application requires those exact endpoints there is nothing we can do about it to make sure it works for you.

We've something in mind to overcome this issue but and is in roadmap; however some limitations may still be applied.

@marcofilippi FYI
